projects/github-actions/required-review/src/request-review.js

```diff
@@ -0,0 +1,66 @@
+const core = require( '@actions/core' );
+const github = require( '@actions/github' );
+const { WError } = require( 'error' );
+
+/**
+ * Fetch Username based on @user
+ *
+ * @param {string} user - @ followed by a GitHub user name.
+ */
+async function getUsername( user ) {
+    const octokit = github.getOctokit( core.getInput( 'token', { required: true } ) );
+    try {
+        const res = await octokit.rest.users.getByUsername( { username: user.slice( 1 ) } );
+        return res.data.login
+    } catch ( error ) {
+        throw new WError(
+            // prettier-ignore
```

Comment:
I see you've ignored Prettier here and across the file in general. Could you remove the exception and run it, so the style remains consistent with the other files in the action?